cpp/tests/db/find_matches_test.cu

```diff
@@ -137,12 +137,11 @@ TEST_F(Test_FindMatches, thirdTest) {
   int32_t* frontier_ptr;
   cudaMalloc(&frontier_ptr, sizeof(int32_t));
```

Comment:
Better avoid using cudaMalloc. And if this is absolutely necessary, guard this with CUDA_TRY.